src/dir.targets

```diff
@@ -94,8 +94,18 @@
   <Import Project="$(ToolsDir)publishtest.targets" Condition="Exists('$(ToolsDir)publishtest.targets')" />
 
   <PropertyGroup>
+    <IsInnerLoopTestProject Condition="'$(IsInnerLoopTestProject)'==''">True</IsInnerLoopTestProject>
```

Comment:
I was hoping that we would have a more general category property. Something like `<TestProjectCategory>InnerLoop;OuterLoop</TestProjectCategory>` and then we could just write conditions against those to do the filtering. I fully expect to get other test categories like Perf, Stress, etc.